src/Components/Actions/ActionsTabItem.js

```diff
@@ -1,32 +1,39 @@
-import React from 'react';
-import './ActionsTabItem.css';
-import hamburgerMenuIcon from 'Images/hamburgerMenuIcon.png';
-import xIcon from 'Images/xIcon.png';
-import { Draggable } from 'react-beautiful-dnd';
+import React from "react";
+import "./ActionsTabItem.css";
+import hamburgerMenuIcon from "Images/hamburgerMenuIcon.png";
+import xIcon from "Images/xIcon.png";
+import { Draggable } from "react-beautiful-dnd";
+import Countdown from "../../Utils/GeneralComponents/Countdown";
 
 export default function ActionsTabItem({ actionItem, index }) {
+  const duration = Countdown(actionItem.startingTime, actionItem.endingTime);
+
   const actionAbortButton = (
     <img
       src={xIcon}
       alt=""
       className="actionsTabItemXIcon smallIcons"
-      onClick={() => console.log('action abort triggered at an actionTabItem')}
+      onClick={() => console.log("action abort triggered at an actionTabItem")}
     />
   );
 
-  const actionEndingTime = Math.floor(actionItem.endingTime / 100000);
-  //actionItem.endingTime is a unix timestamp regarding it's end,
-  //the division and floor are just for the large int to be small enough to fit in the screen
-  //Need to create dynamic counter for this and replace it.
-
   return (
-    <Draggable draggableId={actionItem.id} index={index}>
-      {provided => (
-        <div className="actionsTabItem" {...provided.draggableProps} ref={provided.innerRef}>
-          <img src={hamburgerMenuIcon} alt="" className="smallIcons" {...provided.dragHandleProps} />
+    <Draggable id={actionItem.id} draggableId={actionItem.id} index={index}>
+      {(provided) => (
+        <div
+          className="actionsTabItem"
+          {...provided.draggableProps}
+          ref={provided.innerRef}
+        >
+          <img
+            src={hamburgerMenuIcon}
+            alt=""
+            className="smallIcons"
+            {...provided.dragHandleProps}
+          />
           <div>{actionItem.title}</div>
-          {' - '}
-          <div>{actionEndingTime}</div>
+          {" - "}
+          {duration}
```

Comment:
we didn't get around to it, but the div around {duration} is necessary for the spacing of the elements.
due to the `justify-content: space-between;` that exists at `ActionsTabItem.css/.actionsTabItem`
I'm sure there's a better way to do it, but I'm strategically not getting myself into CSS until odeya's test for `styled-components`